rector_examples_updated/file_modify_permissions.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Drupal\Core\File\FileSystemInterface;
```

Comment:
@Kbentham, can you check if you are using rector 0.7.*? When I updated to that, I stopped getting `use` statements. We want them, but it wasn't working for me.

Reply:
I am on rector version v0.7.16

Reply:
Thanks. The mysteries of the way this `use` statement is or isn't added continues to puzzle me.